### ISSUE TITLE
Building the conda package fails

### ISSUE THREAD

**Issue by Midnighter** [conda-forge/optlang-feedstock#23]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

Quite possibly due to a dated versioneer version that is not being maintained, the current build fails.

### Installed packages

```shell
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=812853&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3
```

### Environment info

```shell
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=812853&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3
```

**Comment by Midnighter** [comment]
@cdiener an option could be to replace versioneer with setuptools-scm. I don't really feel like doing so, but the whole package setup could be modernized in the same go.

**Comment by cdiener** [comment]
Oh yeah probably the same issue as https://github.com/biosustain/swiglpk/pull/88 . Updating versioneer should fix that. I'll prep a patch.

**Comment by cdiener** [comment]
Should be fixed with 1.8.1.
